docs/postpaid/response-code.md

```diff
@@ -2,52 +2,52 @@
+20 | PRODUCT UNREGISTERED | Failed | Your inputted product_code isn’t in the database. Check again your product_code, you can check product_code list by using pricelist API (https://api.iak.id/docs/reference/ad7437ff00ecc-price-list).
+30 | PAYMENT HAVE TO BE DONE VIA COUNTER / PDAM | Failed | Your inputted payment has to be done at the counter. Please do your transaction at the counter.
+31 | TRANSACTION REJECTED DUE TO EXCEEDING MAXIMAL TOTAL BILL ALLOWED | Failed | Your current transaction is exceeding the maximal total bill allowed. Please try to reduce your transaction.
+32 | TRANSACTION FAILED, PLEASE PAY BILL OF ALL PERIOD | Failed | Your current transaction isn’t covering all periods. Please try again to pay with all periods.
+33 | TRANSACTION CAN'T BE PROCESS, PLEASE TRY AGAIN LATER | Failed | There is an error on your body field. Please check again your product code for typos, or retry with other product code.
+34 | BILL HAS BEEN PAID | Failed | Your bill for your current transaction has been paid. Please try again for another transaction.
+35 | TRANSACTION REJECTED DUE TO ANOTHER UNPAID ARREAR | Failed | Your transaction id failed. Please pay for your other arrear first, then try again your transaction.
+36 | EXCEEDING DUE DATE, PLEASE PAY IN THE COUNTER / PDAM | Failed | Your current transaction is exceeding the due date. Please pay the transaction at the counter.
+37 | PAYMENT FAILED | Failed | Your payment failed, please try again later.
+38 | PAYMENT FAILED, PLEASE DO ANOTHER REQUEST | Failed | Your payment failed, please try again with a new request.
+39 | PENDING / TRANSACTION IN PROCESS | Pending | Your transaction is still being processed. Please wait for further status updates.
+40 | TRANSACTION REJECTED DUE TO ALL OR ONE OF THE ARREAR/INVOICE HAS BEEN PAID | Failed | Your transaction has already been paid. You can try again with another transaction.
+41 | CAN'T BE PAID IN COUNTER, PLEASE PAY TO THE CORRESPONDING COMPANY | Failed | Your current transaction cannot be paid in the counter. You can pay your transaction to the corresponding company.
+42 | PAYMENT REQUEST HAVEN'T BEEN RECEIEVED | Failed | Your current transaction is still in the inquiry process. Please continue your payment process first.
+91 | DATABASE CONNECTION ERROR | Failed | There is an error on the database connection. Please try again later.
+92 | GENERAL ERROR | Failed | The received response code is undefined yet. Please contact our Customer Service.
+93 | INVALID AMOUNT | Failed | The amount inputted isn’t valid or identical with the amount from the supplier. Please check again your inputted amount.
+94 | SERVICE HAS EXPIRED | Failed | -
+100 | INVALID SIGNATURE | Failed | Your sign (signature) field doesn’t contain the right key for your current request. Please check again your sign value.
```

Comment:
Your sign field doesn’t contain the right key for your current request. Please check again your sign value.